unity/PitayaExample/Assets/Pitaya/PitayaMetrics.cs

```diff
@@ -0,0 +1,172 @@
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.IO;
+using Google.Protobuf;
+using UnityEngine;
+using System.Linq;
+
+namespace Pitaya
+{
+    public class PitayaMetrics
+    {
+        public ConnectionSessionStats connectionSessionStats;
+        public delegate void MetricsCallback(ConnectionSessionStats stats);
+        private Stopwatch connectionWatch = new Stopwatch();
```

Comment:
We generally write private variables with a leading underscore
```suggestion
        private Stopwatch _connectionWatch = new Stopwatch();
```